crates/astria-sequencer/src/mempool/mod.rs

```diff
@@ -360,46 +315,43 @@ mod test {
     use astria_core::crypto::SigningKey;
 
     use super::*;
-    use crate::app::test_utils::get_mock_tx_parameterized;
+    use crate::app::test_utils::mock_tx;
 
     #[tokio::test]
     async fn insert() {
         let mempool = Mempool::new();
         let signing_key = SigningKey::from([1; 32]);
 
         // sign and insert nonce 1
-        let tx1 = get_mock_tx_parameterized(1, &signing_key, [0; 32]);
+        let tx1 = mock_tx(1, &signing_key, "test");
         assert!(
             mempool.insert(tx1.clone(), 0).await.is_ok(),
             "should be able to insert nonce 1 transaction into mempool"
         );
 
         // try to insert again
-        assert!(
-            matches!(
-                mempool.insert(tx1.clone(), 0).await,
-                Err(InsertionError::AlreadyPresent)
-            ),
+        assert_eq!(
+            mempool.insert(tx1.clone(), 0).await.unwrap_err(),
+            InsertionError::AlreadyPresent,
             "already present"
         );
 
         // try to replace nonce
-        let tx1_replacement = get_mock_tx_parameterized(1, &signing_key, [1; 32]);
-        assert!(
-            matches!(
-                mempool.insert(tx1_replacement.clone(), 0).await,
-                Err(InsertionError::NonceTaken)
-            ),
+        let tx1_replacement = mock_tx(1, &signing_key, "test");
+        assert_eq!(
+            mempool
+                .insert(tx1_replacement.clone(), 0)
+                .await
+                .unwrap_err(),
+            InsertionError::AlreadyPresent,
```

Comment:
To preserve the original assert intent:
```suggestion
        let tx1_replacement = mock_tx(1, &signing_key, "test_0");
        assert_eq!(
            mempool
                .insert(tx1_replacement.clone(), 0)
                .await
                .unwrap_err(),
            InsertionError::NonceTaken,
```